tests/integration/views/test_questionnaire.py

```diff
@@ -535,3 +535,42 @@ def test_updating_questionnaire_store_removes_completed_block_for_calculation_de
             update_questionnaire_store_with_form_data(self.question_store, calculation_answer_location, calculation_answer_data)
 
         self.assertNotIn(dependent_location, self.question_store.completed_blocks)
+
+    def test_updating_questionnaire_store_specific_group(self):
+        g.schema = load_schema_from_params('labour', 'force')
```

Comment:
I don't think it's a good idea to tie our tests to live questionnaire schemas. Let's keep them restricted to test schemas.

Perhaps test_repeating_household could be augmented to include this scenario?

If you need to you can create a minimal test schema inline for the test and create a `QuestionnaireSchema` from it. See `test_only_question_blocks_counted_for_completeness` in `test_completeness.py` for an example. I think augmenting `test_repeating_household` is probably better in this case though.

Reply:
Good points - will take another look

Reply:
Looked at augmenting `test_repeating_household` schema but the conditions that give rise to the issue arise specifically when you have a question that can be skipped and if so route to a question that is normally routed "around".  So rather than a fair amount of fiddling with it and obscuring the purpose of tests that currently use it, I think it would be better to use a separate test schema and add a new module `integration/household_composition/` using new schema.  Objections/thoughts?